mission/para_server_init.sqf

```diff
@@ -196,37 +365,126 @@ call vn_mf_fnc_group_init;
     } forEach (getArray (_taskConfig >> 'taskGroups'));
 } forEach (vn_mf_secondaryTaskConfigs);
 
-// start generic scheduler functions
+/*
+=========================================================================================
+job: `save_profiledb`
+=========================================================================================
+Write the contents of `profileNamespace` to the $profile.vars file, i.e. flush data from
+memory to disk for persistance between server restarts.
+
+Disk write ops are the slowest of all data storage ops. Please avoid setting the job
+schedule time to a low number! Lower periodic job runs means writing to disk more often.
+
+Seeting lower job times *probably* has performance impacts. I'm not 100% sure as I don't
+know the internals of Arma3 and whether they set a 'lock' on the data in memory while
+writing. Locks -- https://en.wikipedia.org/wiki/Lock_(computer_science)
+
+This might not be required as the `saveProfileNamespace` command is called automatically
+'when the game is closed'. But i don't know whether 'when the game is closed' includes
+server restarts, which basically are a termination of the running Arma3 process.
+https://community.bistudio.com/wiki/saveProfileNamespace
+=========================================================================================
+*/
+
+diag_log "VN MikeForce: Initialising 'save_profiledb' job.";
+["save_profiledb", {["SAVE"] call para_s_fnc_profile_db}, [], 60] call para_g_fnc_scheduler_add_job;
+
```

Comment:
new job saving profile data to disk once a minute instead of every 5 seconds (running inside the old `save_time_elapsed` job)

Reply:
see comment above the code line for explanation about why this was factored out